ScnSoft-Education/LeetCodeTraining/RemoveDuplicatesFromSortedArray-26/RemoveDuplicatesFromSortedArraySolution.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace LeetCodeTraining.RemoveDuplicatesFromSortedArray
+{
+    //https://leetcode.com/problems/remove-duplicates-from-sorted-array/
+    public class RemoveDuplicatesFromSortedArraySolution
+    {
+        public static int RemoveDuplicates(int[] nums)
+        {
+            if (nums.Length == 0)
+            {
+                return 0;
+            }
+
+            int k = 1;
+
+            for (int i = 1; i < nums.Length; i++)
+            {
+                if (nums[i] != nums[i - 1])
+                {
+                    nums[k] = nums[i];
+                    k++;
```

Comment:
Please use k++ in previous line.

Reply:
By using k++ in previous line meant:

`nums[k++] = nums[i];`

Because as it is now the code is not valid and doesn't work properly.
Please fix.